tutorials/grid_topology.ipynb

```diff
@@ -50,19 +50,11 @@
    "execution_count": 2,
```

Comment:
<p>Can we avoid this error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1071/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Sorry, notebook outputs and git aren't the best of friends.